### DISCUSSION TITLE
4.4.69 install EXE on VirusTotal - 3 Malware hits

### DISCUSSION THREAD

**Discussion by Objectivite** [Open-Shell/Open-Shell-Menu#826] [General]
Ran the OpenShell 4.4.69 exe on  www.VirusTotal.com  and 3 antivirus engines flagged it as having malware.

O.

**Comment by Ibuprophen** [comment]
This is unfortunately a False Positive that's a common occurance in that website for this software. 

Thank you very much for bringing this to our attention anyways. :-)) 


~Ibuprophen
